namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/hmpps-registers.tf

```diff
@@ -0,0 +1,12 @@
+resource "kubernetes_secret" "hmpps-registers" {
+  metadata {
+    name      = "hmpps-domain-events-topic"
+    namespace = "hmpps-registers-dev"
```

Comment:
pushing creds to other namespace

Reply:
@Mjwillis , I dont think you can push creds secrets to another namespace as they share different terraform states.

Reply:
yes you can @poornima-krishnasamy

Reply:
We already do this in offender-events-dev namespace

Reply:
Thats interesting @Mjwillis. Thanks for the reference.